pkg/controller/job/job_controller.go

```diff
@@ -448,6 +448,16 @@ func (jm *Controller) deletePod(logger klog.Logger, obj interface{}, final bool)
 	jm.enqueueSyncJobBatched(logger, job)
 }
 
+func (jm *Controller) addJob(logger klog.Logger, obj interface{}) {
+	jm.enqueueSyncJobImmediately(logger, obj)
+	jobObj, ok := obj.(*batch.Job)
+	if ok {
```

Comment:
Nit for readability: 
```golang
jobObj, ok := obj.(*batch.Job)
if !ok {
    return
}
if controllerName := managedByExternalController(jobObj); controllerName != nil {
    metrics.JobByExternalControllerTotal.WithLabelValues(*controllerName).Inc()
}
```
Fast return allows you to limit the amount of indentations one has to parse :wink:

Reply:
sure, done